core/graph/README.md

```diff
@@ -54,4 +54,104 @@ to be run before and after an operation chain is executed.
 
 You can use hooks to do things like custom logging or special operation
 chain authorisation. To implement your own hook, just implement the `GraphHook`
-interface and register it with the graph when you build a `Graph` instance.
\ No newline at end of file
+interface and register it with the graph when you build a `Graph` instance.
+
+## FAQs
+To use an example of the authorised Graph exporter within the road-traffic-demo using the proxy-store, follow these steps.
+
+-Clone Gaffer twice, instance A and instance B.
+
+-On A:
+
+1.  Add below to road-traffic-demo/pom.xml:
+```
+<dependency>

```

Comment:
To help with the formatting you can tell markdown the type of code block like this:

```xml
<dependency>
    <groupId>uk.gov.gchq.gaffer</groupId>
    <artifactId>proxy-store</artifactId>
    <version>${project.parent.version}</version>
 </dependency>
```